src/main/kotlin/no/nav/familie/ba/mottak/søknad/SøknadController.kt

```diff
@@ -0,0 +1,21 @@
+package no.nav.familie.ba.mottak.søknad
+
+import main.kotlin.no.nav.familie.ba.søknad.Søknad
+import no.nav.familie.kontrakter.felles.Ressurs
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import no.nav.security.token.support.core.api.Unprotected
+import org.springframework.http.ResponseEntity
+import org.springframework.http.MediaType.APPLICATION_JSON_VALUE
+import org.springframework.http.MediaType.MULTIPART_FORM_DATA_VALUE
+import org.springframework.web.bind.annotation.*
+import java.time.LocalDateTime
+
+@RestController
+@RequestMapping(path = ["/api"], produces = [APPLICATION_JSON_VALUE])
+@ProtectedWithClaims(issuer = "selvbetjening", claimMap = ["acr=Level4"])
+class SøknadController {
+    @PostMapping(value= ["/soknad"], consumes = [MULTIPART_FORM_DATA_VALUE])
```

Comment:
Er vel Henning som har bedt om dette, men.
Er ikke regelen at vi generelt ikke skriver kode for ting som kommer i fremtiden, men for hvordan de er nå? For øyeblikket sender vi jo ingen vedlegg, så da burde vi vel heller ha 
consumes = [APPLICATION_JSON_VALUE]?

Reply:
Det kommer i en senere iterasjon, så lar den stå atm